releasenotes/notes/fastapi-fix-middlewares-705975a535daaea8.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    fastapi: Ensure spans are created for all user middlewares.
```

Comment:
We don't actually generate a span for each middleware do we? Would it be more accurate to say "Ensure traced middleware span duration encompasses all middlewares"? Or something like that?